platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/command/ListApplicationCommandTest.java

```diff
@@ -34,7 +34,7 @@
 import org.slf4j.LoggerFactory;
 
 public class ListApplicationCommandTest {
-    private Logger logger = LoggerFactory.getLogger(ListApplicationCommand.class);
+    private Logger LOGGER = LoggerFactory.getLogger(ListApplicationCommand.class);
```

Comment:
We're starting to get out of scope here, but it doesn't look like this LOGGER is being used in the class